dali/operators/decoder/host/fused/host_decoder_slice.h

```diff
@@ -29,16 +29,13 @@ class HostDecoderSlice : public HostDecoder {
   inline ~HostDecoderSlice() override = default;
   DISABLE_COPY_MOVE_ASSIGN(HostDecoderSlice);
 
- protected:
-  inline void RunImpl(Workspace &ws) override {
+  using HostDecoder::RunImpl;
```

Comment:
Is this needed?

Reply:
Yes, otherwise clang is complaining about hiding a function from the superclass.